src/components/Label/Label.stories.js

```diff
@@ -1,53 +1,49 @@
 /* @flow */
 import React from 'react';
+import R from 'ramda';
+
 import { storiesOf, action, linkTo } from '@kadira/storybook';
 import {
-   withKnobs,
-   text,
-   boolean,
-   number,
-   select,
+  withKnobs,
+  text,
+  boolean,
+  number,
+  select,
 } from '@kadira/storybook-addon-knobs';
 
+import { PositionEnum, SizeEnum } from '../../assets/constants';
+const positionOptions = R.invertObj(PositionEnum);
+const sizeOptions = R.invertObj(SizeEnum);
+
 import Label from '.';
 
 const stories = storiesOf('Label', module);
 stories.addDecorator(withKnobs);
 stories.addDecorator(story => (
```

Comment:
ok never mind - we can keep the adding-margin-decorators... but it would be cool to see the labels in action applied next to a styled square div.